databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,51 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
+      required: false,
+      arrayElementsValidator: {
+        mustNotBeEmpty: true,
+        type: 'object',
+        required: true,
+        propertyValidators: {
+          // indicates what type of annotation this is
+          // Only embedded is supported to start
+          type: {
+            type: 'string',
+            immutable: true,
+            predefinedValues: [ 'embedded' ],
+          },
+          // a simple tag describing what type data this annotation is
+          // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+          dataType: {
+            type: 'string',
+            immutable: true,
+            predefinedValues: [ 'user-metadata' ]
```

Comment:
this may even be overly restrictive, do we really want/expect to have to update this list when we introduce new types?